Notify/NotificationService.swift

```diff
@@ -71,45 +72,34 @@ class NotificationService: UNNotificationServiceExtension {
                         self.bestAttemptMutex.lock {
                             
                             let content = UNMutableNotificationContent()
+                            let (noteSubtitle, noteBody) = unsealedRequest.notificationDetails()
                             
-                            var errorMessage:String?
+                            content.subtitle = noteSubtitle
                             
-                            // approved
+                            // auto-approved
```

Comment:
I wouldn't call this auto-approved -- isn't it just cached (but potentially rejected)?